eslint.config.mjs

```diff
@@ -6,7 +6,6 @@ import { globalIgnores } from 'eslint/config';
 import importX from 'eslint-plugin-import-x';
 import mocha from 'eslint-plugin-mocha';
 import noFloatingPromise from 'eslint-plugin-no-floating-promise';
-import noOnlyTests from 'eslint-plugin-no-only-tests';
```

Comment:
This is covered by `eslint-plugin-mocha`, specifically the `mocha/no-exclusive-tests` rule.

Reply:
Linking so we don't lose any features -- it doesn't look like https://github.com/vitest-dev/eslint-plugin-vitest?tab=readme-ov-file#rules has a direct equivalent (for #11943 )

Reply:
Sure it does, it's just named something different: https://github.com/vitest-dev/eslint-plugin-vitest/blob/main/docs/rules/no-focused-tests.md

Reply:
Ah.